src/uwtools/drivers/forecast.py

```diff
@@ -256,4 +299,93 @@ def _mpi_env_variables(self, delimiter: str = " ") -> str:
         return delimiter.join([f"{k}={v}" for k, v in envvars.items()])
 
 
-CLASSES = {"FV3": FV3Forecast}
+class MPASForecast(Forecast):
+    """
+    A Driver for the MPAS Atmosphere forecast model.
+    """
+
+    def __init__(
+        self,
+        config_file: str,
+        dry_run: bool = False,
+        batch_script: Optional[str] = None,
+    ):
+        """
+        Initialize the Forecast Driver.
+        """
+
+        super().__init__(config_file=config_file, dry_run=dry_run, batch_script=batch_script)
+
+    def create_namelist(self, cycle: datetime, output_path: OptionalPath) -> None:
+        """
+        Uses an object with user supplied values and an optional namelist base file to create an
+        output namelist file. Will "dereference" the base file.
+
+        :param output_path: Optional location of output namelist.
+        """
+        self._create_user_updated_config(
+            config_class=NMLConfig,
+            config_values=self._config["namelist"],
+            output_path=output_path,
+        )
+        start_time = cycle.strftime("%Y-%m-%d_%H:%M:%S")
+        date_values = {
+            "nhyd_model": {
+                "config_start_time": start_time,
+            },
+        }
+        logging.info(f"Updating namelist date values to start at: {start_time}")
+        config_obj = NMLConfig(output_path)
+        config_obj.update_values(date_values)
+        config_obj.dump(output_path)
+
+    def create_streams(self, output_path: OptionalPath) -> None:
+        """
+        Create the streams file from a template.
+        """
+
+        template_file = self._config["streams"]["template"]
+        values = self._config["streams"]["vars"]
+
+        with readable(template_file) as f:
+            template_str = f.read()
+
+        template = J2Template(values=values, template_str=template_str)
+        with writable(output_path) as f:
+            print(template.render(), file=f)
```

Comment:
The streams files are XML, and I'm currently treating them as templates to make it fairly straightforward. We may consider switching that approach at some point.